static/css/components/list--view-body.less

```diff
@@ -0,0 +1,9 @@
+/**
+ * List View Body
+ */
+
+/* stylelint-disable selector-max-specificity */
+#listResults > li:last-child {
+  border-bottom: 1px dotted @lightest-grey;
+}
+/* stylelint-enable selector-max-specificity */
```

Comment:
I think that this can be removed.  The style change is barely perceptible when I toggle this rule in devtools.

Reply:
Yep, it's a very small change  😅  I'll remove it.